docs/TroubleShooting.md

```diff
@@ -54,6 +54,17 @@ make
 sudo make install
 ```
 
+### Issues with toolchain
+When ```make ``` executes, you may encounter below error:
+```
+arm-none-eabi-gcc: Command not found
+```
+This may be because the toolchain you set uses 32-bit libs on the 64-bit system. For this matter, install the below toolchain as alternative.
+To resolve:
```

Comment:
How about changing to new line?
You can add two spaces at the end of the line 62.

Reply:
Done.